src/sst/elements/vanadis/inst/regfile.h

```diff
@@ -52,6 +52,7 @@ class VanadisRegisterFile
     void init( ) {
         std::memset(int_reg_storage, 0, (int_reg_width * count_int_regs));
         std::memset(fp_reg_storage, 0, (fp_reg_width * count_fp_regs));
+        std::fill_n(counters, sizeof(counters), 0);
```

Comment:
@feldergast  Is fill_n supported by our current set of compilers (ranges::fill_n looks like c++-20)?

Reply:
The ranges version would be 20, but this is the iterator/size version (where iterator here is the decayed pointer to the C-array), and I think that's been around forever.